docs/user-guide/how-to.md

```diff
@@ -677,6 +691,10 @@ The most important type of configuration is the way each object type
 
 * **Propagate:** propagates objects from ancestors to descendants and deletes
   obsolete descendants.
+* **AllowPropagate:** inclusive propagation - only propagates objects from ancestors 
+  to descendants and deletes obsolete descendants when
+  [selectors](#limit-the-propagation-of-an-object-to-descendant-namespaces) are set
+  on the object.
```

Comment:
I might be a bit more precise here: "when at least one selector is set on the object."

Reply:
Will change it